plugins/source/aws/go.mod

```diff
@@ -133,9 +134,10 @@ require (
 // TODO: remove once all updates are merged
 replace github.com/apache/arrow/go/v13 => github.com/cloudquery/arrow/go/v13 v13.0.0-20230509053643-898a79b1d3c8
 
+replace github.com/cloudquery/plugin-sdk/v3 => ../../../../plugin-sdk
```

Comment:
This needs to be updated

Reply:
I guess we can only do it once https://github.com/cloudquery/plugin-sdk/pull/864 is merged/released